crates/libs/bindgen/src/rust/extensions/impl/Foundation/Collections/Iterable.rs

```diff
@@ -85,6 +88,6 @@ where
     type Error = windows_core::Error;
     fn try_from(values: Vec<T::Default>) -> windows_core::Result<Self> {
         // TODO: should provide a fallible try_into or more explicit allocator
-        Ok(StockIterable { values }.into())
+        Ok(windows_core::ComObject::new(StockIterable { values }).into_interface())
```

Comment:
As above, be nice if this simple case had a shorthand.

Reply:
It still works; just being explicit.  I can change this back, but I like the explicitness.